packages/core/src/devtools/store.ts

```diff
@@ -0,0 +1,39 @@
+import { SequenceHash } from '../types';
+import { SEQUENCE_PREFIX, SEQUENCE_SIZE } from '../constants';
+import { mergeClassesCachedResults } from '../mergeClasses';
+
+const sequenceDetails: Record<SequenceHash, { slotName: string }> = {};
+
+const cssRules: string[] = [];
+
+export const MK_DEBUG = {
```

Comment:
```suggestion
export const debugData = {
```

Let's use something more meaningful for name (opened for other options)